test/dlc_tests/conftest.py

```diff
@@ -306,6 +301,18 @@ def terminate_ec2_instance():
     ec2_utils.check_system_state(instance_id, system_status="ok", instance_status="ok", region=region)
     return instance_id, key_filename
 
+def is_neuron_image(fixtures):
```

Comment:
nit: should this be named is_neuron_fixture?

Reply:
will do a fast follow in a separate PR